docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -99,6 +99,9 @@ values. Typical usage does not require you to configure them.
       - Caching duration for active files which correspond to the Delta Lake
         tables.
       - ``30m``
+    * - ``delta.metadata.cache-size``
```

Comment:
Can you put it next to `delta.metadata.cache-ttl`? The two refer to the same cache

Reply:
good idea.. the table should probably be alphabetically sorted .. but that can be in a follow up PR